libs/blocks/merch-card/merch-card.js

```diff
@@ -428,8 +428,8 @@ const init = async (el) => {
   }
   merchCard.appendChild(footer);
 
-  if (cardType === 'plans') {
-    const quantitySelect = createQuantitySelect(el);
+  if (['plans', 'product'].includes(cardType)) {
```

Comment:
make of `['plans', 'product']` a constant?

Reply:
OFFER_SELECT_CARD_TYPES ?

Reply:
or MULTI_OFFER_CARDS